src/Core/src/LifecycleEvents/Android/AndroidLifecycle.cs

```diff
@@ -11,10 +11,13 @@ public static class AndroidLifecycle
 		// Events called by the Application
 		public delegate void OnApplicationCreating(Application application);
 		public delegate void OnApplicationCreate(Application application);
-		public delegate void OnApplicationLowMemory(Application application);
-		public delegate void OnApplicationTrimMemory(Application application, TrimMemory level);
 		public delegate void OnApplicationConfigurationChanged(Application application, Configuration newConfig);
 
+		// Events called by IComponentCallbacks2
+		public delegate void OnConfigurationChanged(Configuration newConfig);
```

Comment:
Does this still fire on activity events? Maybe rotation or keyboard things?

Reply:
Yes it works for a changing the orientation even if you don't add `ConfigChanges.Orientation` to `ConfigurationChanges` in your Activity. 


What should I do to have `OnConfigurationChanged` called by keyboard? It is never called in a activity, even if you use this attribute:

```csharp
[Activity(ConfigurationChanges = ConfigChanges.KeyboardHidden | ConfigChanges.Keyboard)]
```